src/note.ts

```diff
@@ -83,6 +83,11 @@ namespace music {
         const pitch: string = _frequencyToMinecraftPitch(note);
         player.execute(`playsound ${soundId} @a ~ ~ ~ ${music.volumeInGameUnits} ${pitch}`)
 
+        // use == here to check for null and undefined
+        if (millis == undefined) {
+            loops.pause(beat(BeatFraction.Whole));
+        }
+
         if (millis > 0) {
```

Comment:
Maybe make this an else if? And same for the one below.

Reply:
I used Thomas' suggestion of changing what millis was to insure that we're not possibly pausing twice.